C++/Navio/Navio2/Led_Navio2.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Led_Navio2.h"
+
+Led_Navio2::Led_Navio2()
+{
+
+}
+
+bool Led_Navio2::initialize()
+{
+    return (!pwm.initialize());
```

Comment:
We don't use PWM for Leds on Navio2.
It's also extremely confusing to negate the initialize()'s return value here. Why?